packages-tests/BetterPhpDocParser/PhpDocInlineHtml/Source/InlineHtmlRector.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Rector\Tests\BetterPhpDocParser\PhpDocInlineHtml;
+
+use PhpParser\Node;
+use PHPStan\PhpDocParser\Ast\PhpDoc\VarTagValueNode;
+use PHPStan\PhpDocParser\Ast\Type\IdentifierTypeNode;
+use Rector\Core\Rector\AbstractRector;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use ViewScopeRector\Inferer\Rocket\FileLocator;
+use ViewScopeRector\Inferer\Rocket\ViewFileLocator;
+
+class InlineHtmlRector extends AbstractRector
+{
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('Defines a @var on inline html.', [new CodeSample('', '')]);
+    }
+
+    public function getNodeTypes(): array
+    {
+        return [Node\Stmt\InlineHTML::class];
+    }
+
+    /**
+     * @param Node\Stmt\InlineHTML $inlineHtml
+     */
+    public function refactor(Node $inlineHtml): ?Node
+    {
+        if ($inlineHtml->getAttribute(AttributeKey::PREVIOUS_NODE) === null) {
```

Comment:
the actual problem I am running into (initial report) seems only be relevant, when the `InlineHTML` node is the very first statement within the file rector is beeing run on.

after the unit-testcase got more stable - I feel this IF should move into `PhpDocInfoPrinter`